runbotics/runbotics-orchestrator-ui/src/hooks/useInitiatorLabel.ts

```diff
@@ -0,0 +1,32 @@
+import { IProcessTrigger, IUser, ProcessTrigger } from 'runbotics-common';
+
+import useTranslations from './useTranslations';
+
+const isProcessTrigger = (
+    trigger: IProcessTrigger | ProcessTrigger
+): trigger is ProcessTrigger => typeof trigger === 'string' && trigger in ProcessTrigger;
+
+const useInitiatorLabel = () => {
+    const { translate } = useTranslations();
+
+    const mapInitiatorLabel = (
+        user: IUser, trigger: IProcessTrigger | ProcessTrigger, triggeredBy: string | undefined
```

Comment:
maybe we can use an object here and type the whole object instead of the parameters